test/fixtures/output/flink/artifact/create-help.golden

```diff
@@ -9,10 +9,11 @@ Create Flink artifact "my-flink-artifact".
   $ confluent flink artifact create my-flink-artifact --artifact-file plugin.jar
 
 Flags:
-      --artifact-file string   REQUIRED: Flink artifact JAR file.
-      --description string     Description of Flink artifact.
-      --context string         CLI context name.
-  -o, --output string          Specify the output format as "human", "json", or "yaml". (default "human")
+      --artifact-file string      REQUIRED: Flink artifact JAR file or ZIP file.
+      --runtime-language string   Specify the Flink artifact runtime language as "python" or "java". (default "java")
```

Comment:
Can I upload a ZIP file with java code? I'm wondering if we can infer the language from the file extension since JAR corresponds to Java.

Reply:
Awesome, sounds like we can remove `--runtime-language` for the time being if that's ok!

Reply:
> you can't. we should update description to imply jar -> java; zip -> python.
btw, backend does throw 400 errors when type mismatch

Currently we can, let me know @byang-confluent  if I am doing something wrong:
```
go run cmd/confluent/main.go flink artifact create cli-test-artifact-v3 --artifact-file ~/Downloads/rushi-test-custom-compute-connect-datagen-0.6.5.zip --description "checking if it works" --runtime-language java
+----------------+----------------------+
| Name           | cli-test-artifact-v3 |
| Plugin ID      | ccp-devclq3np7       |
| Version ID     | ver-devc41jy54       |
| Content Format | ZIP                  |
+----------------+----------------------+
```

Reply:
after this PR is merged https://github.com/confluentinc/cc-connect-plugin-service/pull/687/files

Reply:
Awesome, sounds like we can remove --runtime-language for the time being if that's ok! 
| for now, yes. but it's better to add it as optional input as Rushi did so we don't have to update cli in the future.